File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.js
/**
1. 공백으로 문자열을 나눈다.
2. 나눈 문자열을 저장.
3. 저장한 데이터 중 최댓값과 최솟값을 출력한다.
*/
function solution(s) {
    let array = [];
    for(let i = 0 ; i < s.split(" ").length; i ++){
        array.push(parseInt(s.split(" ")[i]));
    }
    const max = Math.max(...array)
    const min = Math.min(...array)
    
    
    
    return `${min} ${max}`;
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.kt
class Solution {
    fun solution(s: String): String {
        val stringArray = s.split(" ")
        val intArray = IntArray(stringArray.size)
        stringArray.indices.forEach{intArray[it] = stringArray[it].toInt()}
        intArray.sort()
        return "${intArray.first()} ${intArray.last()}"
    }
}